packages/fiber/src/core/index.tsx

```diff
@@ -273,8 +273,9 @@ function createRoot<TCanvas extends Element>(canvas: TCanvas): ReconcilerRoot<TC
 
       // Safely set color management if available.
       // Avoid accessing THREE.ColorManagement to play nice with older versions
-      if ('ColorManagement' in THREE) {
-        setDeep(THREE, legacy, ['ColorManagement', 'legacyMode'])
+      if (ColorManagement) {
+        if ('enabled' in ColorManagement) ColorManagement.enabled = legacy
+        else if ('legacyMode' in ColorManagement) ColorManagement.legacyMode = legacy
```

Comment:
Note the values taken by "enabled" and "legacyMode" would be inverted -- enabled=true is equivalent to legacyMode=false.